drivers/at86rf2xx/periph/hwrng.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2016 FH Campus Wien
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_at86rf2xx
+ * @{
+ *
+ * @file
+ * @brief       Low-level random number generator driver implementation
+ *
+ * @author      Mathias Tausig <mathias.tausig@fh-campuswien.ac.at>
+ *
+ * @}
+ */
+
+#include "at86rf2xx.h"
+#include "at86rf2xx_params.h"
+#include "at86rf2xx_registers.h"
+#include "at86rf2xx_internal.h"
+#include "periph/hwrng.h"
+
+static at86rf2xx_t dev;
+
+void hwrng_init(void)
+{
+  const at86rf2xx_params_t *p = &at86rf2xx_params[0];
+  at86rf2xx_init(&dev,
+		 p->spi,
+		 p->spi_speed,
+		 p->cs_pin,
+		 p->int_pin,
+		 p->sleep_pin,
+		 p->reset_pin);
```

Comment:
OK, so I'll add a `at86rf2xx_get_random` function to at86rf2xx_internal which pull the random values from the register.
Are you aware, if this can be done for all at86rf2xx chips, or do I have to ifdef-restrict it to the at86rf233 (which is the only one I have a documentation for)?